Sadot/User/UserHomePage.cs

```diff
@@ -204,5 +285,146 @@ private void UpdateStockAndOrderStatus(int tableId)
             tableOrder.IsPaid = true;
             db.UpdateOrder(tableOrder);
         }
+
+        /// <summary>
+        /// Timr event wich updates the table list every timer tick
+        /// </summary>
+        private void updateTableListTimer_Tick(object sender, EventArgs e)
+        {
+            FillTableList();
+        }
+
+        /// <summary>
+        /// method wich Create table buttons objects, initilaze and assigned them to main panel controls
+        /// </summary>
+        public void CreateTableButtons()
+        {
+            for (int i = (int)ETableIDs.TABLE_ID_0; i < (int)ETableIDs.MAX_TABLES; i++)
```

Comment:
I think it should be int and not enum (as I said before), but if you are using an enum, you can do:
**foreach (var tableId in Enum.GetValues(typeof(ETableIDs)))**
Just make sure that MAX_TABLES is not included with if or something (talk to me on the phone about it)

Reply:
Fixed - i decided to remove the enum as you suggested